static/scripts/homework.js

```diff
@@ -61,7 +61,10 @@ window.addEventListener("DOMContentLoaded", function(){
     if(!filterModule){return;}
     filterModule.addEventListener('newFilter', (e) => {
         const filter = e.detail;
-        const newurl = "?filterQuery=" + escape(JSON.stringify(filter[0]));
+        const newurl = `?filterQuery=${
```

Comment:
what happens for undefined p value? btw. the function has a refactor/deprecated notation and for sure there exist a library for this use case like https://api.jquery.com/jquery.param/

Reply:
now using desired function from helper file and checking whether `params.p` actually exists